src/components/WwwFrame/DisclaimersContentful.vue

```diff
@@ -0,0 +1,169 @@
+s<template>
```

Comment:
An extra 's' slipped through here. Surprisingly, the Vue loader didn't complain about it 🤔

Reply:
👀 Damn, good catch, ty